scripts/data/WeatherProvider.js

```diff
@@ -1 +1,17 @@
-import { settings } from "../Settings.js"
\ No newline at end of file
+//import { settings } from "../Settings.js"
+
+let weatherForecast = [];
+
+export const getWeather = () => {
+
+    return fetch(`http: //api.openweathermap.org/data/2.5/forecast?id=524901&appid=e78bff76cd169fd6088b48c66c3fd622`)
```

Comment:
http: //api.openweathermap.org/data/2.5/forecast?id=524901&appid=e78bff76cd169fd6088b48c66c3fd622

should be

http://api.openweathermap.org/data/2.5/forecast?id=524901&appid=${settings.weatherKey}